src/OpenSpaceToolkit/Astrodynamics/Estimator/TLESolver.cpp

```diff
@@ -79,17 +91,12 @@ TLESolver::TLESolver(
 {
     // Setup coordinate subsets for TLE state
     Array<Shared<const CoordinateSubset>> coordinateSubsets = {
-        std::make_shared<CoordinateSubset>("INCLINATION", 1),
-        std::make_shared<CoordinateSubset>("RAAN", 1),
-        std::make_shared<CoordinateSubset>("ECCENTRICITY", 1),
-        std::make_shared<CoordinateSubset>("AOP", 1),
-        std::make_shared<CoordinateSubset>("MEAN_ANOMALY", 1),
-        std::make_shared<CoordinateSubset>("MEAN_MOTION", 1)
+        InclinationSubset, RaanSubset, EccentricitySubset, AopSubset, MeanAnomalySubset, MeanMotionSubset
```

Comment:
```suggestion
        InclinationSubset, RaanSubset, EccentricitySubset, AopSubset, MeanAnomalySubset, MeanMotionSubset,
```
This will make it format nicer.